pkg/controller/job/job_controller.go

```diff
@@ -257,7 +257,7 @@ func (jm *Controller) Run(ctx context.Context, workers int) {
 	logger.Info("Starting job controller")
 	defer logger.Info("Shutting down job controller")
 
```

Comment:
I still see bunch of controllers that need an update in the controller package. Can we do it in the same PR, or do you plan to update them later?

Reply:
Ok, thx!